src/components/breedingDisplay.html

```diff
@@ -72,7 +72,7 @@
     <!-- ko if: App.game.breeding.queueList().length -->
     <div id="hatcheryQueue" class="card-footer p-1" data-bind="foreach: App.game.breeding.queueList()">
       <div class="queuePokemon clickable bg-primary" data-bind="click: function(){ App.game.breeding.removeFromQueue($index()) }">
-        <img src="" data-bind="attr: { src: PokemonHelper.getImage(pokemonMap[$data], App.game.party.alreadyCaughtPokemon(pokemonMap[$data].id, true)) }">
+        <img src="" data-bind="attr: { src: PokemonHelper.getImage(pokemonMap[$data].id, App.game.party.alreadyCaughtPokemon(pokemonMap[$data].id, true), App.game.party.getPokemon(pokemonMap[$data].id).defaultFemaleSprite()) }">
```

Comment:
This code is duplicated a couple of times. You might want to add a `PokemonHelper.getPartyImage()`, which does all the lookups. Maybe just add it on PartyPokemon class? I think it fits better there

Reply:
getImage is not just the sprite from party/Pokédex/breeding but also wild/trainer Pokémon, since they did the same task (getting the sprite) I decided to merge them all into a single function.
I made some changes so I can remove the shiny and gender parameters from some calls in the html files.
Since it's also being used for wild/trainer sprites, I didn't move it yet to PartyPokemon, but it can be done if you want.